pkgs/cupertino_http/example/integration_test/client_test.dart

```diff
@@ -45,7 +46,7 @@ void testClient(Client client) {
       }
       final request = StreamedRequest('POST', uri);
       request.sink.add(data);
-      request.sink.close();
+      unawaited(request.sink.close());
       await client.send(request);
```

Comment:
You could also try
```dart
await (request.sink.close(), client.send(request)).wait;
```
(But the current text is equivalent to what was before, so that's the safer bet.)